.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,56 @@
+name: Bug Report
+description: Thanks for taking the time to fill out this bug report!
+
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is. If applicable, add screenshots or logfiles to help explain your problem.
+    validations:
+      required: true
+  - type: dropdown
+    id: affected-domain
+    attributes:
+      label: EVerest Domain
+      description: Can you specify a specific domain is affected by the issue? Mark multiple if applicable.
+      options:
+        - Authorization
+        - Charge Control
+        - CHAdeMO
+        - Energy Management
+        - Hardware Drivers
+        - ISO15118
+        - Power Supplies
+        - OCPP
+        - OCPP1.6
+        - OCPP2.0.1
+        - Simulation
+        - Slack
+        - Compilation
+        - Utility
+        - Testing
```

Comment:
How about adding `Other` and/or `Unknown`? We may have missed a category, and the reporter might not be able to think in these categories.